server/mcp_server_vod/README.md

```diff
@@ -0,0 +1,64 @@
+# Volcengine MCP
+
+Volcengine VOD 的 Model Context Protocol (MCP) Server 实现
+
+## 项目简介
+
+Volcengine VOD MCP是一个基于[Model Context Protocol](https://github.com/modelcontextprotocol/python-sdk)的 MCP Server，
+它将 Volcengine VOD服务集成到LLM模型上下文中，使大模型能够直接操作和管理VOD资源。
+
+## 功能特点
+
+- 提供多种资源访问接口，便于LLM获取Volcengine VOD服务信息、视频资源等
+- 实现了多个Volcengine功能的工具封装，包括上传媒资，剪辑视频等
+
+## Available Tools
+
+- `get_space_detail`: [get space details](https://www.volcengine.com/docs/4/107689)
```

Comment:
more detail in 50 words to describe the method, nobody care about original doc linkage.